src/components/templates/Camp/components/Map.tsx

```diff
@@ -4,6 +4,7 @@ import { CampRoomData } from "types/CampRoomData";
 import "./Map.scss";
 import { enterRoom } from "../../../../utils/useLocationUpdateEffect";
 import { useUser } from "../../../../hooks/useUser";
+import RoomCard from "./RoomCard";
```

Comment:
Can’t see where this is used, did the build succeed?

Reply:
removed this line